UI/window-basic-settings.cpp

```diff
@@ -3384,6 +3462,90 @@ void OBSBasicSettings::ReloadAudioSources()
 	LoadAudioSources();
 }
 
+void OBSBasicSettings::MultichannelToggled(bool state)
+{
+	if (state)
+	{ //display all channels
+		ui->channelSetup->addItem("2.1");
+		ui->channelSetup->addItem("4.0 Quad");
+		ui->channelSetup->addItem("4.1");
+		ui->channelSetup->addItem("5.1");
+		ui->channelSetup->addItem("7.1");
+		blog(LOG_INFO, "Multichannel ON, bitrates > 320 kbs unlocked");
+	//display all bitrates
+		PopulateAACBitrates({ ui->simpleOutputABitrate,
+				ui->advOutTrack1Bitrate, ui->advOutTrack2Bitrate,
+				ui->advOutTrack3Bitrate, ui->advOutTrack4Bitrate,
+				ui->advOutTrack5Bitrate, ui->advOutTrack6Bitrate });
+	}	
+	else
+	{ 
+		blog(LOG_INFO, "Multichannel OFF");
+	/* 
+	* reset channel and bitrates settings when Multichannel is switched OFF
+	* first reset channel menu and save new setting
+	*/
+		int idx_channel = ui->channelSetup->currentIndex();
+		if (idx_channel >= 2) {
+			ui->channelSetup->setCurrentIndex(1);
+			ui->channelSetup->setProperty("changed", QVariant(true));
+			config_set_string(main->Config(), "Audio", "ChannelSetup",
+					"Stereo");
+		}
+		for (int i = 6; i >= 2; i--)
+			ui->channelSetup->removeItem(i);
+		blog(LOG_INFO, "Channel list reset to mono & stereo");
+
+		/* 
+		* reset audio bitrate for simple and adv mode, update list of bitrates
+		* and save setting
+		*/
+
+		blog(LOG_INFO,"Bitrates restricted to under 320kbs");
```

Comment:
Style:  There should be a single space between "LOG_INFO," and the quoted parameter after it.
```
		blog(LOG_INFO, "Bitrates restricted to under 320kbs");
```